code/game/machinery/doors/airlock.dm

```diff
@@ -1042,7 +1042,7 @@ About the new airlock wires panel:
 				else
 					close(1)
 
-	if(istype(I, /obj/item/crowbar/power))
+	if(ispowertool(I)) //jaws of life and rescue claw
```

Comment:
So if someone is holding the rescue claws mech equipment in their hands, will they be able to use it like the jaws of life?

Reply:
Shouldn't work, because the try_to_crowbar function won't be called if you do that.

Reply:
I've tried it in game just to verify and can confirm it doesn't work like that.